homeassistant/components/ness_alarm/alarm_control_panel.py

```diff
@@ -56,10 +58,10 @@ class NessAlarmPanel(AlarmControlPanelEntity):
         | AlarmControlPanelEntityFeature.TRIGGER
     )
 
-    def __init__(self, client: Client, name: str) -> None:
+    def __init__(self, client: Client, panel_name) -> None:
         """Initialize the alarm panel."""
         self._client = client
-        self._attr_name = name
+        self._attr_name = panel_name if panel_name else "Alarm Panel"
```

Comment:
This addition is not correct. The names for devices and entities shouldn't be set through the configuration flow.

If one wants to change the name of an entity, one can do so, through the entity settings dialog and adjust the name to their liking.

If we'd merge this proposal, one would end up with multiple places to override a name, making it unclear which one wins.

Nevertheless, thanks for being willing to contribute 👍 

../Frenck